tests/translator/input/http_api_existing_openapi.yaml

```diff
@@ -26,11 +26,18 @@ Resources:
   MyApi:
     Type: AWS::Serverless::HttpApi
     Properties:
+      CorsConfiguration:
+        AllowOrigins:
+          - "*"
       DefinitionBody:
         info:
           version: '1.0'
           title:
             Ref: AWS::StackName
+        # when this is specified `CorsConfiguration` property is ignored
```

Comment:
Good question about this. What we need to decide is if `CorsConfiguration` should override, merge, or be ignored if there already exists a configuration in the openapi document.

This brings up the bigger question of how do we do this in other areas of SAM for HTTP APIs. I will need to check on this, but I believe that:
* If a configuration doesn't exist in the openapi document, SAM should add it
* If a configuration exists, SAM should have a set pattern of either ignoring or overriding it

Reply:
As we discussed offline, I think we need to **merge** the `CorsConfiguration` with open api cors extension  header if exists and override by giving preference to values in `CorsConfiguration` property. This way SAM can give the flexibility for template authors to override some of the cors headers mentioned in `CorsConfiguration` property and also will be able to add other/new Cors headers irrespective of SAMs support

Reply:
updated the code to merge and override `CorsConfiguration`

Reply:
We are now ignoring the configurations here if there is a corresponding configuration in `CorsConfiguration` (remove comment)